<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuManage</class>
 <widget class="QWidget" name="MenuManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_order">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>88</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>주문확인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>88</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>110</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>영업시간관리</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>88</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>110</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>메뉴 관리</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>88</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
    </property>
    <property name="text">
     <string>홈</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="menu_tw">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>751</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>751</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>메뉴 추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>메뉴 수정</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>메뉴명</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_le"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>가격</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="price_le"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>메뉴카테고리</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_cate"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>메뉴상태</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_menu_status"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>설명</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="desc_le"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>메뉴 삭제</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>761</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3"/>
  </widget>
  <widget class="QFrame" name="frame_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>741</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>전체</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>판매중</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>판매중지중</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_all_menu_count">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_menu_active_state">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_menu_deactive_state">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
